tests/acceptance/features/webUICreateUser/createUserRequirementCombinations.feature

```diff
@@ -0,0 +1,78 @@
+    And user "guiusr1" should not exist
+    Examples:
+      | password                       | message                                                                                       |
+        # where just one of the requirements is not met
+      | aA1!bB2#cC&d                   | The password is too short. At least 15 characters are required.                               |
+      | aA1!bB2#cNOT&ENOUGH#LOWERCASE  | The password contains too few lowercase letters. At least 4 lowercase letters are required.   |
+      | aA1!bB2#cnot&enough#uppercase  | The password contains too few uppercase letters. At least 3 uppercase letters are required.   |
+      | Not&Enough#Numbers=1           | The password contains too few numbers. At least 2 numbers are required.                       |
+      | Not&Enough#Special8Characters2 | The password contains too few special characters. At least 3 special characters are required. |
+        # where multiple requirements are not met, only the first error message is shown to the user
+      | aA!1                           | The password is too short. At least 15 characters are required.                               |
+      | aA!123456789012345             | The password contains too few lowercase letters. At least 4 lowercase letters are required.   |
+
+  Scenario Outline: user sets their password to a string with valid restricted special characters after being created with an Email address only
+    When the administrator creates a user with the name "user1" and the email "guiusr1@owncloud" without a password using the webUI
+    And the administrator logs out of the webUI
+    And the user follows the password set link received by "guiusr1@owncloud" using the webUI
+    And the user sets the password to "<password>" using the webUI
+    Then the email address "guiusr1@owncloud" should have received an email with the body containing
+      """
+      Password changed successfully
+      """
+    When the user logs in with username "user1" and password "<password>" using the webUI
+    Then the user should be redirected to a webUI page with the title "Files - %productname%"
+    Examples:
+      | password                  |
+      | 15***UPPloweZZZ           |
+      | More%Than$15!Characters-0 |
+
+  Scenario Outline: user tries to set their password to a string with invalid restricted special characters after being created with an Email address only
+    When the administrator creates a user with the name "user1" and the email "guiusr1@owncloud" without a password using the webUI
+    And the administrator logs out of the webUI
+    And the user follows the password set link received by "guiusr1@owncloud" using the webUI
+    And the user sets the password to "<password>" using the webUI
+    Then a set password error message with this text should be displayed on the webUI:
+      """
+      The password contains too few numbers. At least 3 numbers are required.
```

Comment:
I am surprised that this will pass. I am expecting the text in the ``message`` column of the examples table